plugins/package-managers/maven/src/main/kotlin/utils/MavenSupport.kt

```diff
@@ -76,32 +75,23 @@ import org.eclipse.aether.transfer.TransferEvent
 import org.eclipse.aether.util.repository.JreProxySelector
 
 import org.ossreviewtoolkit.analyzer.PackageManager
-import org.ossreviewtoolkit.downloader.VcsHost
```

Comment:
Any plans for the remaining `MavenSupport`, renaming split-up?

Reply:
I've added 3 more commits as indeed there was more to clean up.